libkbfs/prefetcher_test.go

```diff
@@ -272,7 +276,106 @@ func TestPrefetcherDirectDirBlockAlreadyCached(t *testing.T) {
 	t.Log("Shutdown the prefetcher and wait until it's done prefetching.")
 	<-q.Prefetcher().Shutdown()
 
-	block, err = cacheFunc().Get(dir2.Children["b"].BlockPointer)
+	block, err = config.BlockCache().Get(dir2.Children["b"].BlockPointer)
 	require.NoError(t, err)
 	require.Equal(t, file1, block)
 }
+
+func TestPrefetcherNoPrefetchWhileCacheFull(t *testing.T) {
+	t.Log("Test that prefetches aren't triggered when the cache is full.")
```

Comment:
No need, just making sure I understand.  I think just clarifying in the log message is good enough for me, but more tests are always welcome of course.